x-pack/plugins/apm/public/components/shared/TransactionDurationAlertTrigger/index.stories.tsx

```diff
@@ -19,7 +19,7 @@ import { MockUrlParamsContextProvider } from '../../../context/UrlParamsContext/
 // our current storybook setup has core-js-related problems when trying to import
```

Comment:
I don't think this is true anymore. Making a note to revisit once #75357 is merged.